applications: nrf_desktop: Remove modules/Kconfig.ble file (#10188)
nrfconnect/sdk-nrf: applications/nrf_desktop/doc/ble_latency.rst

*************

The module requires the basic Bluetooth configuration, as described in :ref:`nrf_desktop_bluetooth_guide`.
Make sure that both :ref:`CONFIG_DESKTOP_ROLE_HID_PERIPHERAL <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_BT_PERIPHERAL <config_desktop_app_options>` options are enabled.

> The Bluetooth LE latency application module is enabled by the :ref:`CONFIG_DESKTOP_BLE_LATENCY_ENABLE <config_desktop_app_options>` option, which is implied by :ref:`CONFIG_DESKTOP_BT_PERIPHERAL <config_desktop_app_options>` together with other features used by a HID peripheral device.

Reviewer: ```suggestion
The Bluetooth LE latency application module is enabled by the :ref:`CONFIG_DESKTOP_BLE_LATENCY_ENABLE <config_desktop_app_options>` option. 
The option is implied by :ref:`CONFIG_DESKTOP_BT_PERIPHERAL <config_desktop_app_options>` together with other features used by an HID peripheral device.
```
